scripts/EN/Harry

```diff
@@ -1916,9 +1916,9 @@ LoadSecondEffect(0x0, 0x0, 0x0); // 0x21f4: 0000306e
 CleanScreen(); // 0x21fe: 00000064
 LoadBackground(0x16); // 0x2202: 00001069
 DrawBackground(); // 0x2208: 00000065
-ShowText((1092)"My one shot hits the wing, blasting a large hole in it. The creature loses altitude as it falls to the floor."); // 0x220c: 0000100c
+ShowText((1092)"My one shot hits the wing, W1blasting a large hole in it.¥nW1The creature then lost altitude and fell to the floor with a smack."); // 0x220c: 0000100c
 HandleInput(); // 0x2212: 00000010
-ShowText((1093)"I can't afford to waste bullets. I crush it with the with the thick sole of my shoe many times. As its strange cries die out, a dark red stain spreads out on the floor."); // 0x2216: 0000100c
+ShowText((1093)"I can't afford to waste bullets.¥nW2I stomp on it with the thick sole of my shoe many times.¥nW2As its strange cries die out, W1a dark red stain spreads out on the floor."); // 0x2216: 0000100c
```

Comment:
Removed an extra instance of "with the". Thanks to Hector for catching this typo!

Also changed "crush" to "stomp on" because that suits both the English and the Japanese better.